### DISCUSSION TITLE
editable layers cross platform behavior

### DISCUSSION THREAD

**Discussion by vaduga** [visgl/deck.gl-community#101] [General]
hello, 
I was wondering if someone could give me a clue of why editable-geojson-layers may not show features on some platforms, while deck.gl base layers are visible under the same conditions. For example:
- in android browsers editable geojson points do show up, but not the lines features. Basic deck.gl layers show up just fine.
- after migrate to v.9, in addition to android users,  ios safari browser also started to not show neither points nor lines in editable geojson points, while base geojson deck.gl layers show features fine.

Thanks to the contributors for the great tool anyway!

**Comment by ibgreen** [comment]
@vaduga Thanks for reporting. I don't think we have done any testing on mobile. It is not a primary use case for editable-layers as screen sizes tend to be a bit limited for editing. Are you able to extract any logs or provide any other information? E.g. which specific layer in the editable-layers module is not working?

**Comment by vaduga** [comment]
probably it is for some ios-specific reason, as on ios 18 it all works fine again. 
android issues of not showing linestring features remain, but i don't have any significant sampling
